### PR TITLE
Gauri development

### PR DESCRIPTION
**Pull request by Gjain234**
-I added support for a basic admin page only accessible by url for now. can be reached using the /admin url. 

-it will currently work if your username is beckyqiu or gaurijain so try making a fake account with those or try it with other accounts to see how it denies access. Ernesto/Maria I will update it with your usernames shortly, just lmk which names you want me to include!

-lastly I'm trying to figure out the best way to test whether the jsp is outputting the correct thing on the page based on what your username is. What would be the way to do that? 

Here are the UI screenshots!

screen shot 2018-05-29 at 11 30 36 pm
screen shot 2018-05-29 at 11 29 03 pm

### CODE REVIEW

File: src/test/java/codeu/controller/AdminServletTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class AdminServletTest{
```
Comment (beckyqiu) [inline src/test/java/codeu/controller/AdminServletTest.java]: nit: Empty space needed after class name.

**Review by ernesto-rojas — COMMENTED** [review]
Hey, this looks great! I am curious about the "denied" text that is displayed when a non-admin attempts to access the admin page. Moving forward, would this behavior continue to exist? I guess I am asking whether non-admins should be aware of the existence of the admin page in the first place? Do my questions make sense? If not, feel free to ask for clarification!

**Review by mrios730 — COMMENTED** [review]
Hi Gauri! The code looks good. Do you know how you will be able to gather the statistics of the app? Also, will you be adding more tests to your AdminServletTest file as you move forward? You can use mariarios as my username :)

**Comment by Gjain234** [comment]
@TF3FAN so thats a good point. Currently what I'm thinking is maybe we add a tab to the toolbar for the admin page but that tab will only show up if an admin logs in. Currently the only way to access the page is to use the link, so if for some bizarre reason a non-admin got ahold of the admin link, we would just deny access. Let me know what you think! This is actually something I was still unsure about design wise as well.

@mrios730 So for now I'm thinking of just making a function that can return the number of users from the UserStore class or the number of conversations from the Conversations class. I have the getSize() working for users and will do conversations soon! Do you have ideas for other things that might be good to include for the MVP? And yeah I'll definitely try to add more tests here as we go. I wasn't sure how to test the super basic page beyond just making sure it reached the webpage, but I should hopefully have more soon.

**Review by beckyqiu — APPROVED** [review]
Please work on the MVP after this one submitted. :)

**Pull request merged**